src/utils/presets.ts

```diff
@@ -4512,6 +4527,8 @@ export const PRESET_CALENDARS: Calendar[] = [
             months: [
                 {
                     name: "Hammer (Deepwinter)",
```

Comment:
I am not sure I like keeping the subtitle in the name. I think it might make more sense to have:

```ts
{
  name: "Hammer",
  subtitle: "Deepwinter"
  ...
}
```
and build the rest out from there.

We can keep the short name, which people can use to specify the abbreviated month name (e.g. Feb, like you said).

Thoughts?